src/scss/_header.scss

```diff
@@ -0,0 +1,121 @@
+$team-background-color: $primary-color;
```

Comment:
Not sure why this file is called `_header.scss`, also I thought separating css folders was in another tasks. Do not include other tasks into this PR, it is already large as is. Please move this code back into `App.scss`